nemo_rl/models/policy/dtensor_policy_worker.py

```diff
@@ -65,6 +65,7 @@
     get_gpu_info,
     get_runtime_env_for_policy_worker,
     import_class_from_path,
+    is_vllm_v1_engine_enabled,
```

Comment:
I'm not a huge fan of having inference backend-specific stuff here. Can we make this passed into here via some other mechanism or do you feel this is fine?

Reply:
I realize it's just an envvar check, but this is a weird logical dependency for the user ("what does 'v1' or 'v0' in vllm mean? and why is it modifying my logprobs?")